ui/components/app/currency-input/currency-input.js

```diff
@@ -178,11 +196,7 @@ export default function CurrencyInput({
     let suffix, displayValue;
 
     if (!isFiatAvailable || !tokenToFiatConversionRate) {
-      return (
-        <div className="currency-input__conversion-component">
-          {t('noConversionRateAvailable')}
-        </div>
-      );
+      return null;
```

Comment:
Not related to this line - can we disable the Confirm button when Swap + send-ing with 0 amount? It's supported for simple Send (useful for cancelling txs, etc), but I don't see a usecase for using a 0-amount swap trade

Reply:
Still seeing it locally. I'm able to click on Confirm, but it doesn't do anything. Doesn't attempt to fetch quotes either

![Screenshot 2024-05-14 at 4 44 32 PM](https://github.com/MetaMask/metamask-extension/assets/100321200/9b484aed-ac62-4112-86ca-323967b3dff1)

Reply:
Oh my bad I thought you meant the error state – generalized now: b4ebf6e41dab6861384c0af82db6ab6a03953a58